### ISSUE TITLE
DONKEYTOWN needs a CURSE SYSTEM

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLifeData7#1194]
There are a lot of DONKEYS in DONKEYTOWN.

/die players that end up here should be able to curse other players so that they are not born to them.

Similarly, we need to be able to see curse names, and when those cursed donkeys are born, they should very far away.

**Comment by Shadie116** [comment]
pz jason

**Comment by Shadie116** [comment]
i agree, really the game just needs to fix cursing. Donkey town is arguably worse than it was without the biomes

**Comment by Shadie116** [comment]
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/7a189388-d950-46be-922e-6d8a72d9411f)
could have fooled me 🙄 fooled my entire group
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/07365e02-2201-4f07-ad6c-29e5029cf709)

**Comment by Shadie116** [comment]
you permabanned an entire group of 7 people

**Comment by Shadie116** [comment]
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/9e3ce5d1-22e6-401a-b6ca-f6f2c9a9b0f6)
also you banned my friend who ain't did nothing wrong, she left the game entirely.

**Comment by Shadie116** [comment]
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/b1397f51-16df-497b-a805-075d0de9fb2b)

**Comment by Shadie116** [comment]
now lets see my score after a peirod of about a year
![image](https://github.com/jasonrohrer/OneLifeData7/assets/126846097/3cf4b3c6-5e73-4b27-a83e-bb2d404511f7)

**Comment by Tarr-OHOL** [comment]
Curse names were removed from donkey town because reformed and his buddy basically blacklisted any of the babies that weren’t their buddies. Donkey town serves to let even the cruddiest players at least be able to play the game. 

What kind of idiot would actually sit around and be raised a donkey if they’re just a /die baby? You can literally /die back to the normal game and repeat your cycling.

Also cursing doesn’t block you from having the troll as your mom, it blocks them from being born to you/around you. This thread lacks actual knowledge of mechanics and should be closed.

**Comment by Shadie116** [comment]
reformed didn't do nothing lol, nobody knows who did that

**Comment by Shadie116** [comment]
reason we used curse names was to kill racists and or mainly people who were counterproductive to our cause like what you're doing right now in the main game. The people who'd kill the sheep, ruin our radios and all that. As griefers we were actually organized with planning and methods as well as morals IE. No breaking monuments. I can't blame you for dehumanizing your enemies though because that's just how it is, if you had Insite into what happens than things would be alot more clear.

**Comment by DougLefelhocz** [comment]
> What kind of idiot would actually sit around and be raised a donkey if they’re just a /die baby?  You can literally /die back to the normal game and repeat your cycling.

This post comes from forman who has multiple forum posts about things like not having new families to play in, or wanting someway to play from the beginning (and Github issues also).  Perhaps often, he can't do that with other people on bs2, and everyone else on low pop servers deliberately checks a custom server.  So, I suspect he goes to DonkeyTown for that sort of early game experience.  Also, it's not likely that someone with a truck will soon bring resources to a camp 3 hours after an Eve has died.

**Comment by CommunityExile** [comment]
> Curse names were removed from donkey town because reformed and his buddy basically blacklisted any of the babies that weren’t their buddies. Donkey town serves to let even the cruddiest players at least be able to play the game.
> 
> What kind of idiot would actually sit around and be raised a donkey if they’re just a /die baby? You can literally /die back to the normal game and repeat your cycling.
> 
> Also cursing doesn’t block you from having the troll as your mom, it blocks them from being born to you/around you. This thread lacks actual knowledge of mechanics and should be closed.

Why don't you shut up about donkey town politics that don't affect you
